connections/sim/sim_connection.py

```diff
@@ -107,20 +111,29 @@ def _handleBuzzer(self):
     def _handleDigitalPinWrite(self):
         length = self._getLength()
         assert length == 2
-        [pin, value] = self.stdout.read(2)
+        pin, value = self.stdout.read(2)
 
+        self._hw_sim.digital_write(pin, value)
         print("SIM: Pin %s set to %s" % (pin, value))
 
     def _handleRadio(self):
         length = self._getLength()
         data = self.stdout.read(length)
         self._xbee.recieved_from_rocket(data)
 
+    def _handleAnalogRead(self):
+        length = self._getLength()
+        assert length == 1
+        pin = self.stdout.read(length)[0]
+        result = self._hw_sim.analog_read(pin).to_bytes(2, "big")
+        self._send_sim_packet(b"a", result)
```

Comment:
Can we please make the `b"a"` a constant so that people don't get confused about the magic value?